### PR TITLE
72 update custom apps to be compatible with new launch sequence

### PR DESCRIPTION
**Pull request by BPerlakiH**
- Update the common story board for custom apps.
- Update their info.json data, to include the welcomeLogo width and height.
- Update the README

### CODE REVIEW

File: src/info_parser.py [hunk 1]
```diff
@@ -58,7 +64,6 @@ def as_project_yml(self):
         dict = {
             "templates": ["ApplicationTemplate"],
             "settings": {"base": {
-                # TODO: change to .semantic, once builds are OK
```
Comment (BPerlakiH) [inline src/info_parser.py:61]: This comment is no longer needed.

**Comment by BPerlakiH** [comment]
The DWDS app also needs the loading text localised (DE):
```
"welcome.loading.data.text" = "Loading Data...";
```

**Comment by BPerlakiH** [comment]
This is good to go, in cooperation with https://github.com/kiwix/kiwix-apple/pull/1008 gives the desired outcome for custom apps.

**Comment by kelson42** [comment]
@BPerlakiH Image width and height should be retrieved from the image file itself. This is fully redondant and source of error to have to specify it in the json.

**Comment by BPerlakiH** [comment]
> @BPerlakiH Image width and height should be retrieved from the image file itself. This is fully redondant and source of error to have to specify it in the json.

@kelson42 Yes, you are right. I have updated it so we get the size during runtime here: https://github.com/kiwix/kiwix-apple/pull/1019

**Pull request merged**